playwright/src/tests/api.spec.ts

```diff
@@ -39,6 +39,19 @@ test.describe("API endpoints", () => {
             await expect(board.getNote("Updated item")).toBeVisible()
         })
 
+        await test.step("Update new item", async () => {
```

Comment:
There's quite a bit of logic related to handling the partial parameters. I suggest adding a test step that verifies that the item's properties have actually been updated. You could test that by getBoard API (see the BoardApi test helper).

Reply:
yep, need to work on test cases